src/Libraries/Microsoft.Extensions.AI.Evaluation.Console/Commands/ReportCommand.cs

```diff
@@ -17,17 +20,37 @@ namespace Microsoft.Extensions.AI.Evaluation.Console.Commands;
 internal sealed partial class ReportCommand(ILogger logger)
 {
     internal async Task<int> InvokeAsync(
-        DirectoryInfo storageRootDir,
+        DirectoryInfo? storageRootDir,
+        Uri? endpointUri,
         FileInfo outputFile,
+        bool openReport,
         int lastN,
         Format format,
         CancellationToken cancellationToken = default)
     {
-        string storageRootPath = storageRootDir.FullName;
-        logger.LogInformation("Storage root path: {storageRootPath}", storageRootPath);
+        IResultStore resultStore;
 
-        var results = new List<ScenarioRunResult>();
-        var resultStore = new DiskBasedResultStore(storageRootPath);
+        if (storageRootDir is not null)
+        {
+            string storageRootPath = storageRootDir.FullName;
+            logger.LogInformation("Storage root path: {storageRootPath}", storageRootPath);
+
+            resultStore = new DiskBasedResultStore(storageRootPath);
+        }
+        else if (endpointUri is not null)
+        {
+            logger.LogInformation("Azure Storage endpoint: {endpointUri}", endpointUri);
+
+            var fsClient = new DataLakeDirectoryClient(endpointUri, new DefaultAzureCredential());
```

Comment:
Wondering if this may be insufficient like we saw with the doc team's use case where they also had to supply Tennant Id... Not much we can do for now though - we can revisit if we hear feedback.